### PR TITLE
Remove additional Pinniped instructions from TKG docs

### PR DESCRIPTION
**Pull request by antgamdia**
### Description of the change

Once Kubeapps 2.3.3 is out, installing an additional Pinniped concierge version is no longer required. This PR is to remove those instructions from the existing step-by-step docs.
Also, minor auto-formatting changes.

### Benefits

No misunderstandings for people using the latest version of Kubeapps.

### Possible drawbacks

N/A

### Applicable issues

N/A

### Additional information

N/A

### CODE REVIEW

File: docs/step-by-step/kubeapps-on-tkg/step-1.md [hunk 1]
```diff
@@ -2,50 +2,11 @@
...
 
-As per TKG v1.3.1, the built-in Pinniped version is relatively old (v0.4.1) and has some incompatibilities with the current Kubeapps releases. Specifically, Kubeapps requires Pinniped v0.6.0 or higher. However, Pinniped can be installed multiple times as a standalone product (not as a _TKG addon)_ in separate namespaces and using different Kubernetes API groups.
-
-> **NOTE**: Although users can install Pinniped in any namespace with any API group, due to current limitations in Kubeapps, the list of supported combination of namespace/API groups is limited to:
->
-> - `*.pinniped-dev` resources in the `pinniped-concierge` namespace.
-> - `*.pinniped.tmc.cloud.vmware.com` in the `vmware-system-tmc` namespace.
->   However, note that, even though the `tmc` name appears in the above namespace, it is only required for technical reasons. There is no requirement to use TMC when running Kubeapps on TKG.
```
Comment (antgamdia) [inline docs/step-by-step/kubeapps-on-tkg/step-1.md:15]: I think we should create a subsection or sth for TMC users, but I still have to figure out the best approach here.
Perhaps a section replacing the snippets with the TMC ones and just references to that section when appropriate in the rest of the sections. I don' know

Comment (absoludity) [reply docs/step-by-step/kubeapps-on-tkg/step-1.md:15]: What happens currently if a user registers their TKG cluster with TMC? They get two pinniped's running? If so, yes, we can just note that and point to some other TMC-specific docs if they want to configure the TMC pinniped instead? But +1 to think about this a bit more and do it separately.

**Review by absoludity — APPROVED** [review]
It's marked as draft, but guessing that just hasn't been updated since the release. +1

File: docs/step-by-step/kubeapps-on-tkg/step-1.md (hunk 1 above)
Comment (antgamdia) [reply docs/step-by-step/kubeapps-on-tkg/step-1.md:15]: Yes, another Pinniped instance is installed (in the `vmware-system-tmc` ns), but I don't know with is the preference here: whether using the TMC or the TKG Pinniped. I guess people using TMC would like to use this one (for authenticating to multiple clusters using their existing account).

So, I agree, let's just write a note for now (but let's revisit it if any TMC-related question pops up)

Comment (vrabbi) [reply docs/step-by-step/kubeapps-on-tkg/step-1.md:15]: The usual case i havr seen is actually still using the tkg provided pinniped and not the tmc one. I have yet to see a customer of tkg that uses the tmc pinniped

**Pull request merged**